frontend/openchat-shared/src/domain/identity.ts

```diff
@@ -150,19 +151,6 @@ export type FinaliseAccountLinkingResponse = { kind: "success" } | AccountLinkin
 
 export type AccountLinkingError = {
```

Comment:
Is this still used? I think you can delete it. It is exactly the same as `OCError`.